### ISSUE TITLE
cycleway vs. cycleway:both

### ISSUE THREAD

**Issue by mangerlahn** [openstreetmap/id-tagging-schema#1025]
The iD Tagging schema currently uses `cycleway` to define whether a given way features cycling infrastructure of any kind. 
Relating tags are `cycleway:left` and `cycleway:right`. `cycleway:both` exists alongside `cycleway`, with mostly the same meaning. `cycleway:both` is seeing wide adoption in recent years and recently surpassed usage of `cycleway` (1.1 million vs. 1 million uses). According to the [wiki entry](https://wiki.openstreetmap.org/wiki/Key:cycleway:both) for `cycleway:both`, this tag is preferable over `cycleway` as its definition is more clear and consistent. 

I would suggest that the key `cycleway` used in `data/fields/cycleway.json` should be replaced with `cycleway:both`, as it appears to be the more popular of the two by now.
This would also make it more consistent with similar behaving tags like `parking:both`.

Not sure if ways using `cycleway` should offer a migration. The tag appears to be still valid, so a deprecation seems to be premature.

Any opinion on this?

**Comment by mangerlahn** [comment]
Just realized this is a dupe of #743. Argument is even stronger now that `cycleway:both` is more common than `cycleway`.

**Comment by tordans** [comment]
I vote to leave this open and consider it a different solution than https://github.com/openstreetmap/id-tagging-schema/issues/743. In https://github.com/openstreetmap/id-tagging-schema/issues/743 the idea is to expand the logic to handle both.

This issue could be about switching the default-both-tag from `cycleway` to `cycleway:both`. Wicht, AFAIK, is what https://github.com/openstreetmap/id-tagging-schema/pull/1026 does with minimal changes. Once the PR-Preview is there we can test this in a few places.

**Comment by tordans** [comment]
@mangerlahn personally I would happily +1 a proposal to official deprecate the "null" case in favor of the "both" case which would help a lot to add the other tagging to the deprecation file which will trigger a tag-update-notice. And we could also run MapRoulette Challenges to update the "null" case tagging.

**Comment by mangerlahn** [comment]
A deprecation/migration would have to be done carefully though, as cycleway would sometimes not refer to both left and right. 
Otherwise I’m all for it!

**Comment by 1ec5** [comment]
> A deprecation/migration would have to be done carefully though, as cycleway would sometimes not refer to both left and right.

Yes, for example, many mappers have used the unsuffixed `cycleway` key on both sides of a dual carriageway. This doesn’t necessarily mean there’s a cycleway in the median, although that does sometimes happen too. I’m not sure if the usual deprecation mechanism in iD can handle such nuances.

**Comment by tordans** [comment]
@mangerlahn @1ec5 I started writing a proposal to clean up this situation. Please help improve it at https://wiki.openstreetmap.org/wiki/Proposal:Make_cycleway:both_the_default_to_indicate_both_sides 

I will wait for a few weeks so we can iterate on the document before considering next steps.

**Comment by tordans** [comment]
I started the RFC on my proposal https://community.openstreetmap.org/t/rfc-feature-proposal-make-cycleway-both-the-default-to-indicate-both-sides/116585 – Please write a supportive comment over there in case this proposal makes sense to you. It will help to move this forward.
